run_dir/static/js/reads_total.js

```diff
@@ -77,10 +77,16 @@ function update_all_totals(){
         ar_clusters.push(checked);
         if (w_q30>75){
             data[0].data.push(checked);
+            data[1].data.push(0);
+            data[2].data.push(0);
         }else if (w_q30>30){
             data[1].data.push(checked);
+            data[0].data.push(0);
+            data[2].data.push(0);
         }else{
             data[2].data.push(checked);
+            data[0].data.push(0);
+            data[2].data.push(0);
```

Comment:
Anyway, I did `git reset` to remove what was not supposed to be here. Should be ok now?